framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -280,6 +284,10 @@ XYDelaunayGenerator::generate()
     poly2tri.elem_type() = libMesh::ElemType::TRI6;
   else if (_tri_elem_type == "TRI7")
     poly2tri.elem_type() = libMesh::ElemType::TRI7;
+  // Add interior points before triangulating. Only points inside the boundaries
+  // will be meshed.
+  for (auto & point : _interior_points)
+    mesh->add_point(point);
```

Comment:
What kind of checks poly2tri will do on these points? Not external to the boundaries? No overlap?

Reply:
I got errors if points overlapped. If the interior points are not within the boundaries they just don't get meshed, but no warning.